website/content/docs/connect/gateways/terminating-gateway.mdx

```diff
@@ -12,7 +12,7 @@ description: >-
 -> **1.8.0+:** This feature is available in Consul versions 1.8.0 and newer.
 
 Terminating gateways enable connectivity within your organizational network from services in the Consul service mesh to
-services outside the mesh. These gateways effectively act as Connect proxies that can
+services/[destinations](/docs/connect/config-entries/service-defaults#destination) outside the mesh. These gateways effectively act as Connect proxies that can
```

Comment:
i think this might be a better link to reference

```suggestion
services and [destinations](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) outside the mesh. These gateways effectively act as Connect proxies that can
```

Reply:
i almost said "it might be a better destination" 😛